Rules/Rule/SorcererStartCardsModifiedRule.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace Rules.Rule
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Boardgame.BoardEntities;
+    using DataKeys;
+    using HarmonyLib;
+
+    public sealed class SorcererStartCardsModifiedRule : RulesAPI.Rule
+    {
+        public override string Description => "Sorcerer start cards are modified";
+
+        public struct Card
+        {
+            public AbilityKey Name;
+            public bool IsReplenishable;
+        }
+
+        private static bool _isActivated;
+        private static List<Card> _cards;
+
+        public SorcererStartCardsModifiedRule(List<Card> cards)
```

Comment:
Could we be passing two arguements to this function, the first one being the name of the Hero to be changing, and the seconf being the list of cards. We don't need to have it all in a single data structure. If we did this then we don't need duplicate classes for adjusting the other heros.

Reply:
I agree 100% 👍🏾!  At the moment, each rule type can appear only once in a ruleset (Ruleset stores rules in a `Hashset`), so we'll have to resolve https://github.com/orendain/DemeoMods/issues/83 before we can convert this rule to one that we could use for multiple heroes.